reference/platform-app-yaml.md

```diff
@@ -396,4 +398,10 @@ crons:
         cmd: "cd public ; drush core-cron"
 ```
 
+## Top level document_roots
+
+Platform.sh requires that the document root not be at the root of the project.  It is important for security that
+private file mounts not be web-accessible. If the document_root is set to `/`, it will move the application to
+`public` and use that as the root to allow for the private mounts to be kept private.
```

Comment:
No, that's wrong. With `web.locations./.root: ""`, the document root will be the root of the application, hence having mounts below it. This is obviously not recommended, but it's possible.